ft/loader/loader.cc

```diff
@@ -1078,7 +1078,7 @@ static void* extractor_thread (void *blv) {
     FTLOADER bl = (FTLOADER)blv;
     int r = 0;
     while (1) {
-        void *item;
+        void *item = NULL;
```

Comment:
nullptr ?

Reply:
Are we building mysql in C++11 mode? I tried to check that, and the only mention of that I found is that mysql8 will be built in c++11 mode. I don't see CMAKE_CXX_STANDARD mentioned anywhere in the repos and --std is only mentioned once with gnu99.

Now that I searched for it, nullptr is mentioned in our codebase, so it does work somehow.

Reply:
PerconaFT is C++11 AFAIK

Reply:
Yes, TokuDB and PerconaFT require c++11